src/Group.php

```diff
@@ -12,8 +12,13 @@
 use function in_array;
 use function is_object;
 
-final class Group implements GroupInterface
+final class Group
 {
+    public const PREFIX = 'prefix';
```

Comment:
Not sure these constants are needed because they are in the middle of autocomplete list when defining routes.